src/azure-cli/azure/cli/command_modules/containerapp/_up_utils.py

```diff
@@ -597,7 +597,16 @@ def run_acr_build(self, dockerfile, source, quiet=False, build_from_source=False
         now = datetime.now()
         tag_now_suffix = str(now).replace(" ", "").replace("-", "").replace(".", "").replace(":", "")
         image_name_with_tag = image_name + ":{}".format(tag_now_suffix)
-        self.image = self.registry_server + "/" + image_name_with_tag
+
+        if ":" in image_name.split("/")[-1]:
```

Comment:
what about image name is a string "image"?

Reply:
![image](https://github.com/user-attachments/assets/1332bb3b-4342-4d0b-b86e-6b91ae53b38e)

>>> image_name="xinyutest"
>>> print(image_name.split("/")[-1])
xinyutest
>>> print(":" in image_name.split("/")[-1])
False